src/components/metrics.tsx

```diff
@@ -84,6 +89,123 @@ import {
 // Stores information about the currently focused query input
 let focusedQuery;
 
+type PredefinedQueryType = {
+  name: string;
+  query: string;
+};
+
+export const PreDefinedQueriesDropdown = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
```

Comment:
I'm going to leave it as a React.useState(). After attempting to switch to the useBoolean() hook the <Select> component couldn't open anymore.

Reply:
After debugging a bit, the `<Select>` from Patternfly 4 fires the `onToggle` function twice when trying to open, leading to the `useBoolean` hook to loop back to the original `closed` state, so the `useState` is fine here. @zhuje added a comment to note this in the codebase in case further actions are needed to be taken on opening the dropdown